### DISCUSSION TITLE
Production use for basic tab navigation

### DISCUSSION THREAD

**Discussion by boblitex** [callstack/react-native-bottom-tabs#90] [Q&A] [answered]
If you have a basic React Native app with bottom tabs, does nothing fancy, just move from tab to tab and stacks in tabs, would you say this is production ready for this use case now?

**Comment by okwasniewski** [comment, marked as answer]
Hey, yes I would say it's production ready (which is not equal to bug free)! But most of the bugs for basic scenario already got fixed. 

Please open an issue if you encounter something so we can get it fixed 👍🏻
